.gitignore

```diff
@@ -104,3 +104,7 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+test/legit.log
```

Comment:
What is this?!

Reply:
I'm working on tests so I tried to avoid adding them

Reply:
If You don't want too add tests in comments just add it to your local .gitignore, but not in pull request .gitignore file. Or You can add just those files that you really need in your commit, so just skip test files when you use `git add` command